LambdaPayroll/GeneratedPayrollScheme.fs

```diff
@@ -66,19 +67,29 @@ let ContractDeductedPersonsCount =
 let ContractIsBasePosition = 
     HrAdmin.readScalarFromDb<Boolean> (ElemCode "ContractIsBasePosition") { TableName = "hr.Contract"; ColumnName = "IsBasePosition" }
 
+type DeductionsRecord = 
+    { RangeStart: System.Decimal;
+      RangeEnd: System.Decimal;
+      Value: System.Decimal;
+      DeductedPersonsCount: System.Decimal }
+
+[<Extension>]
+type DeductionsExtensions =
+    [<Extension>]
+    static member inline RangeStart (a: PayrollElem<DeductionsRecord>) = PayrollElem.map (fun x -> x.RangeStart) a
```

Comment:
Why inline?

Reply:
Inline methods can be used to optimize performance (in this case we have a method with a single line body that delegates to another function, so we can avoid a function call)
However we shouldn't overuse inline to perform manual micro-optimizations unless it is absolutely necessary. This should be left to automatic compiler optimizations.